src/main/fc/cli.c

```diff
@@ -2320,7 +2321,17 @@ static void cliStatus(char *cmdline)
 {
     UNUSED(cmdline);
 
-    cliPrintLinef("System Uptime: %d seconds", millis() / 1000, vbat);
+    cliPrintLinef("System Uptime: %d seconds", millis() / 1000);
+    cliPrint("Current Time: ");
+    date_time_t dt;
+    if (rtc_get_dt(&dt)) {
+        char buf[FORMATTED_DATE_TIME_BUFSIZE];
+        date_time_format(buf, &dt);
+        cliPrint(buf);
```

Comment:
Please replace this with `cliPrintLinef` - this way you won't need a separate call to `cliPrintLinefeed`